src/createServer.js

```diff
@@ -1,3 +1,83 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase.js');
+
+const validateInput = (originalText, targetCase) => {
```

Comment:
Generally when working with servers it's very important to split most of the stuff into separate modules / files. It decouples your applications, enables easier modifications and separation of the business logic.

Ideally you would create a server directory exporting helpers / modules.
![image](https://github.com/mate-academy/node_case-transformer-server/assets/87986955/7a735dfb-f727-4554-b248-6becaa40cd56)
![image](https://github.com/mate-academy/node_case-transformer-server/assets/87986955/649cf383-6f64-4bad-8a00-e18810c8c121)

Reply:
![image](https://github.com/mate-academy/node_case-transformer-server/assets/87986955/9cce098d-2d88-4e0e-8709-a61001642a1a)

Reply:
![image](https://github.com/mate-academy/node_case-transformer-server/assets/87986955/25f2c1ec-46d5-4579-b474-c53c6bf0ed7a)

Reply:
![image](https://github.com/mate-academy/node_case-transformer-server/assets/87986955/97c1c7f2-1388-4577-b778-fdde0c1fd8d4)

Reply:
![image](https://github.com/mate-academy/node_case-transformer-server/assets/87986955/8111cb57-bf40-4ac7-89ae-75ad6567e12e)

Reply:
And finally the implementation of the createServer
![image](https://github.com/mate-academy/node_case-transformer-server/assets/87986955/f1607537-10e4-4798-920b-ed67871c970d)

Reply:
I'm wondering, why should this be so explicit?
![image](https://github.com/mate-academy/node_case-transformer-server/assets/78635631/c0cce2c1-8dc5-4608-af48-ffec8ecf4758)
If I just send statusCode, the default message would be the same.